concreate/schema/image.yaml

```diff
@@ -1,3 +1,10 @@
 # To be merged with common.yaml
 map:
   from: {type: str, required: True}
+  version: {type: text, required: False}
```

Comment:
cannot this be in common schema?

Reply:
probably

Reply:
I've addressed this through an additional commit.